doc/api-changes/db_filters.rst

```diff
@@ -0,0 +1,4 @@
+db.filters
```

Comment:
You have discovered an old `api-changes` folder that we no longer use. (I think Tom dumped some notes here a long time ago, but they aren't included in the published docs. We should remove them to avoid this confusion.) You should add a new bullet point to the bottom of the "User-Facing API Changes" section of `doc/source/whats_new/v0.9.0.txt`